xlive/Blam/Engine/devices/devices.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+#include "Blam/Cache/DataTypes/TagRef.h"
+#include "Blam/Engine/objects/object_definition.h"
+
+enum e_device_definition_flags : int32
+{
+    _device_definition_position_loops = FLAG(0),
+    _device_definition_unused = FLAG(1),
+    _device_definition_allow_interpolation = FLAG(2)
+};
+
+enum e_device_lightmap_flags : int16
+{
+    _lightmap_dont_use_in_lightmap = FLAG(0),
+    _lightmap_dont_use_in_lightprobe = FLAG(1),
+};
+
+
+struct _device_definition
```

Comment:
struct s_defice_definition?

Reply:
I'd do that but that's the data structure name the editing tools reference

Reply:
og bungie typo 🤔